cli/js/dispatch.ts

```diff
@@ -79,55 +85,62 @@ export function setPluginAsyncHandler(
   PLUGIN_ASYNC_HANDLER_MAP.set(opId, handler);
 }
 
+function getAction(opId: number): Action | void {
+  const OP_ACTION_MAP: OPActionMap = {
```

Comment:
Looks like this object is constructed every single time on function invocation which might be a problem.
You can use an array for this actually.

No idea if v8 can do some interesting optimization over switch statements though.

Reply:
Later, I will give the test results of the two methods for comparison